### ISSUE TITLE
Quest progress carries between games?

### ISSUE THREAD

**Issue by Dreamrisen** [talonos2/BlightfallTracker#25]
I logged out of one multiplayer game and into another, and my quest progress was carried over from the previous one. I opened my quest journal and saw that I was in a party with the member from the previous game -- someone who had never appeared on the server I had just logged into.

**Comment by talonos2** [comment]
This is going from one server to another server? Or from a server to a single-player game? Were either of them LAN worlds? If so, who was hosting?

I want to help, but I require more detail.

**Comment by Dreamrisen** [comment]
Oh, my apologies, I thought I added more details. I guess I never hit the submit button.

I was hosting a game over hamachi using the "open to lan" option in minecraft. I was playing with one friend (Vulpixy). We were having fun. We were on team beta in a party named "Team Beta"

Later, I hopped over to a game another friend of mine was hosting. It's one we had already been playing for some time in. Vulpixy was not part of this game, just me and Corntrollio. He was hosting this game, also over hamachi with "open to lan" in minecraft. We were also on team beta in a party named "Team Beta". I was leader of both parties.

When I logged into Corntrollio's world and opened my quest book, I saw that I was in a party alone with Vulpixy, who had never entered this world. I do not know whether or not it affected Corntrollio's quest progress.

I logged out and logged back in, hoping it didn't save my quest data with Vulpixy and overwrite my quest data with Corntrollio. When I logged back in, everything was back to normal. I only wanted to submit this so it was a known bug in case it happened to anyone else, but I appeared to have been able to fix it by relogging immediately.

Is this enough detail? I'm happy to answer questions. You know my contact info, Talonos2.
